### ISSUE TITLE
FNX-5172 ⁃ TalkBack swipe order of bottom browser components is wrong

### ISSUE THREAD

**Issue by eeejay** [mozilla-mobile/fenix#1671]
With the find in page bar open, the current swipe order, starting from the unlabeled site info button in the toolbar is as follows:

1. Toolbar
a. Unlabled button
b. url textview
c. Tabs button
d. Menu button
1. Find in page
a. Find in page edit box
b. Previous result button
c. Next result button
d. Dismiss find in page button
1. Quick actions button - _This is a bug, the quick actions button should not be accessible with find in page is open. Nonetheless, when it is supposed to be visible it should have an earlier focus order than the toolbar_

The visual order from top to bottom is find in page, quick actions button, toolbar. The accessibility swipe order should match that.

**Comment by mcarare** [comment]
Browser Toolbar is not visible anymore when Find in Page is opened and it cannot be accessed by Talkback with this feature opened.
@eeejay: Closing this issue. If you can replicate it again, please feel free to reopen ticket with new STR. TY!
